<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeamBdbTool</class>
 <widget class="QMainWindow" name="TeamBdbTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>531</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="showImage">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Show Image</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGraphicsView" name="graphicsView">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>400</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>640</width>
          <height>480</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QCheckBox" name="showAllFoundLines">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Show all</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QToolButton" name="toolButton">
          <property name="text">
           <string>Choose</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="popupMode">
           <enum>QToolButton::DelayedPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonIconOnly</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QListView" name="allLinesView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="showAllDetetectedLines">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Show all</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QToolButton" name="toolButton_2">
          <property name="text">
           <string>Choose</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QListView" name="selectedLinesView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>578</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <widget class="QMenu" name="menu_ffnen">
     <property name="title">
      <string>Öffnen</string>
     </property>
     <addaction name="actionDatei"/>
     <addaction name="actionOrdner"/>
    </widget>
    <widget class="QMenu" name="menuDatenbank">
     <property name="title">
      <string>Datenbank</string>
     </property>
     <addaction name="action_ffnen_Erstellen"/>
     <addaction name="actionDatenschreiben"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="actionNeu"/>
    <addaction name="separator"/>
    <addaction name="menu_ffnen"/>
    <addaction name="menuDatenbank"/>
   </widget>
   <addaction name="menuDatei"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDatei"/>
   <addaction name="actionOrdner"/>
   <addaction name="separator"/>
   <addaction name="action_ffnen_Erstellen"/>
   <addaction name="actionDatenschreiben"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNeu">
   <property name="icon">
    <iconset resource="../myIcons.qrc">
     <normaloff>:/icons/icons/My-Documents-icon.png</normaloff>:/icons/icons/My-Documents-icon.png</iconset>
   </property>
   <property name="text">
    <string>Neu</string>
   </property>
  </action>
  <action name="actionDatei">
   <property name="icon">
    <iconset resource="../myIcons.qrc">
     <normaloff>:/icons/icons/My-Pictures-icon.png</normaloff>:/icons/icons/My-Pictures-icon.png</iconset>
   </property>
   <property name="text">
    <string>Datei</string>
   </property>
  </action>
  <action name="actionOrdner">
   <property name="icon">
    <iconset resource="../myIcons.qrc">
     <normaloff>:/icons/icons/Folder-Open-icon.png</normaloff>:/icons/icons/Folder-Open-icon.png</iconset>
   </property>
   <property name="text">
    <string>Ordner</string>
   </property>
  </action>
  <action name="action_ffnen_Erstellen">
   <property name="icon">
    <iconset resource="../myIcons.qrc">
     <normaloff>:/icons/icons/database-add-icon.png</normaloff>:/icons/icons/database-add-icon.png</iconset>
   </property>
   <property name="text">
    <string>Öffnen/Erstellen</string>
   </property>
  </action>
  <action name="actionDatenschreiben">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../myIcons.qrc">
     <normaloff>:/icons/icons/database-arrow-down-icon.png</normaloff>:/icons/icons/database-arrow-down-icon.png</iconset>
   </property>
   <property name="text">
    <string>Daten schreiben</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../myIcons.qrc"/>
 </resources>
 <connections/>
</ui>
